packages/mobile/components/upload/src/renderless/index.ts

```
@@ -0,0 +1,488 @@
+  service,
+  props,
+  options,
+  rawFile,
+  state,
+  uid,
+  uploaderInner,
+  api,
+  constants
+}: Pick<IUploadRenderlessParams, 'service' | 'props' | 'state' | 'api' | 'constants'> &
+  IUploadRenderlessOtherParams & { options: IUploadOptionsOfPost }) => {
+  if (service && service.network && props.httpRequest === service.network.request) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)